supervision/annotators/utils.py

```diff
@@ -1,5 +1,5 @@
 from enum import Enum
-from typing import Optional, Tuple, Union
+from typing import Optional, Union
```

Comment:
lets replace `Union` with `|` and `Optional` with `| None` to keep things consistent